docs/BUILD.md

```diff
@@ -122,6 +132,15 @@ specific versions without polluting the user's normal workspace.
 
 The following are the output components from the build process and their purpose. There are two kinds of components: final, intended for actual direct usage, and interim, used to build the final components. Some interim components may be removed as part of the build finalization process.
 
+However, **every** build will create `rootfs.tar`. This contains the root filesystem, whose contents then are
```

Comment:
Every? E.g., make pkg/pillar doesn't create a rootfs.tar

rootfs.tar is always a precursor for creating the live vs. installer.

Reply:
Good point. Every build of a bootable eve image passed through `rootfs.tar`. I will update it.

Reply:
Updated, please take a look.